assertj-core/src/main/java/org/assertj/core/error/ShouldBeInIntegerFormat.java

```diff
@@ -0,0 +1,11 @@
+package org.assertj.core.error;
+
+public class ShouldBeInIntegerFormat extends BasicErrorMessageFactory{  public static ErrorMessageFactory shouldBeInIntegerFormat(Object actual) {
```

Comment:
Add a unit test for this, you can take ShouldBeEven_create_Test as an example to follow

Reply:
reformat the code with `mvn spotless:apply`

Reply:
Thank you for the great comment. I change the files following the comment. 
1. Add the error unit test
2. change assertThat -> BDDAssertions.then
3. use parametized test instead of using pure test 
4. mvn spotless:apply